webpack/assets/javascripts/react_app/components/notifications/toast/Toast.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import Alert from '../../common/Alert';
+
+const Toast = (props) => {
```

Comment:
Not crucial but how about destructuring the props here so that it's clear which props we expect?